### DISCUSSION TITLE
Android 4.2

### DISCUSSION THREAD

**Discussion by tellts** [termux/termux-app#2955] [General]
Hello. I have a 4-core tablet with 1 gigabyte of memory and a diagonal of about 10. Can you please tell me the version of the program that can be installed? The latest generic version is not installed.

**Comment by agnostic-apollo** [comment]
Termux never supported Android 4.2 and only 5+. You may try https://play.google.com/store/apps/details?id=jackpal.androidterm instead

**Comment by TriHydera** [comment]
What not update

**Comment by pressfanksss** [comment]
try to raise the android's response to 9 well or 10, it is better not to take versions above 10, there is a patch that interferes with thermuks and other applications that need root folders (for libraries, distributives, and so on), update do not sit on 4.1

**Reply by pressfanksss** [reply]
4.2*

**Comment by tellts** [comment]
"4.2*": Sorry to bother you, but I didn't understand what you meant.
